Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n# Editor-based HTTP Client requests\r\n/httpRequests/\r\n# Ignored default folder with query files\r\n/queries/\r\n# Datasource local storage ignored files\r\n/dataSources/\r\n/dataSources.local.xml\r\n
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision a7dec62aa9d8cb3ef46bec0ef208e60997ee2192)
+++ b/.idea/.gitignore	(date 1781256344993)
@@ -8,3 +8,4 @@
 # Datasource local storage ignored files
 /dataSources/
 /dataSources.local.xml
+*
\ No newline at end of file
